arc/utils/loops.py

```diff
@@ -254,6 +285,28 @@ def _get_next_run(self) -> float:
             - datetime.datetime.now(self._tz).timestamp()
         )
 
+    def set_interval(self, cron_format: str) -> None:
```

Comment:
I'm not entirely sure this is a good idea in the case of crons specifically, but even if so, then it is missing the timezone parameter.